### DISCUSSION TITLE
我正在学习gkd规则时，遇到的疑问

### DISCUSSION THREAD

**Discussion by heshi2019** [gkd-kit/gkd#751] [Q&A] [answered]
在审查工具中编写完关系表达式后，成功找到了对应的组件

![image](https://github.com/user-attachments/assets/44a767f2-9451-4ca0-a88a-c796ff588eca)

复制规则后我想尝试测试，但提示 ‘非法规则: matches 和 anyMatches 至少存在一个’

![image](https://github.com/user-attachments/assets/8b8f9a94-f1f8-4567-8c75-f1588f6911ac)

请教下这是为什么

快照：https://i.gkd.li/i/14045424
关系表达式：FrameLayout[id="com.netease.cloudmusic:id/container"] >(5) @TextView[name="android.widget.TextView"]

**Comment by lisonge** [comment, marked as answer]
因为你放置的是规则组，这个编辑框目前只能校验单个规则

![image](https://github.com/user-attachments/assets/88bfc2ff-805c-4e63-b554-502797f6b6eb)

---

补充: 已支持校验其他结构的规则组

**Reply by heshi2019** [reply]
感谢
![image](https://github.com/user-attachments/assets/39c4898c-2200-4249-b615-45117fab48d7)
